### DISCUSSION TITLE
Website-Logo Problem

### DISCUSSION THREAD

**Discussion by franz-licht** [GetPublii/Publii#1817] [Help] [answered]
Hello dear Publii community,

I'm Franz and I'm brand new to the fantstic Publii. I have a problem with the website logo. The original size (340px x 43px) of the PNG file is greatly expanded. What am I doing wrong?

Publii 0.46.2
Taste Theme 3.0.0.0
Linux Mint Cinamon 21.2

**Comment by gpsblues** [comment, marked as answer]
Hi, 
try forcing the dimensions with custom CSS.

```
.logo>img {
    height: 43px;
}
```

**Reply by franz-licht** [reply]
Thank you Vittorio, yes it works. I have added the following to center the logo
```
.logo>img {
    height: 22px;
    vertical-align: baseline;
}
```
Greetings to italy! ;)
